indico/modules/events/layout/templates/layout_meeting_lecture.html

```diff
@@ -1,10 +1,23 @@
 {% extends 'layout/base.html' %}
-{% from 'forms/_form.html' import simple_form %}
+{% from 'forms/_form.html' import simple_form, form_rows %}
 
 {% block title %}
     {%- trans %}Layout{% endtrans -%}
 {% endblock %}
 
 {%- block content %}
-    {{ simple_form(form, save_reminder=true, back_button=false) }}
+     {% call simple_form(form, save_reminder=true, back_button=false) %}
+         <div>
+            {{ form_rows(form, fields=('timetable_theme', 'timetable_by_room', 'timetable_detailed')) }}
+            {% if timetable_theme_settings_form %}
+                {{ form_rows(timetable_theme_settings_form) }}
+            {% endif %}
+         </div>
+    {% endcall %}
+    <script>
+        showThemeSettingsForm({
+            formUrl: {{ url_for('.timetable_theme_form', event) | tojson }},
+            lastFieldId: 'form-group-timetable_theme'
```

Comment:
Just FYI, you could get the ID from the form: `'form-group-{{ form.timetable_theme.id }}'`